File: src/main/java/eu/fayder/restcountries/v1/domain/Country.java
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
package eu.fayder.restcountries.v1.domain;

import java.util.List;

import eu.fayder.restcountries.domain.BaseCountry;

public class Country extends BaseCountry {

    private List<String> currencies;
    private List<String> languages;
    private CountryTranslations translations;
    private String relevance;

    public List<String> getCurrencies() {
        return currencies;
    }

    public List<String> getLanguages() {
        return languages;
    }

    public CountryTranslations getTranslations() {
        return translations;
    }

    public String getRelevance() {
        return relevance;
    }
}

File: src/main/kotlin/eu/fayder/restcountries/domain/Country.kt
package eu.fayder.restcountries.domain

data class Country
(
        val currencies: List<Currency>?,
        val languages: List<Language>,
        val flag: String,

        // From mledoze
        val names: Names
//        val translations: Translations,
//        val cioc: String
//        val landlocked: Boolean,
) : BaseCountry()